src/sentry/migrations/0359_duplicate_perf_issue_events_issue_platform.py

```diff
@@ -0,0 +1,222 @@
+
+    for row in rows:
+        try:
+            # don't need to store in node_store since it should be there already, maybe check and write if doesn't exists
+            # create issue occurrence
+            # save issue occurrence
+            event, occurrence_data, group_id = _map_performance_issue_row_to_params(row)
+
+            group: Group = Group.objects.get(id=group_id)
+
+            occurrence, group_info = _save_issue_occurrence(occurrence_data, event, group)
+
+            send_issue_occurrence_to_eventstream(event, occurrence, group_info)
+        except Exception:
+            print("Failed to process row")  # noqa: S002
+
+
+def _map_performance_issue_row_to_params(
+    row: Mapping[str, Any]
+) -> Tuple[Event, IssueOccurrenceData, int]:
+    # we need to do the inverse of TransactionProcessor (Event -> EventStreamPayload -> Row):
+    # Row -> Event
+    # some columns in the rows have been extracted out from the Event interfaces to a top-level row and stripped out
+    # we might need to put them back in
+    data = {
+        "platform": "",
+        "message": "",
+        "tags": [],
+        "breadcrumbs": {},
+        "contexts": {},
+        "extra": {},
+        "logger": "",
+        "metadata": {},
+        "request": {},
+        "spans": [],
+        "transaction": "",
+        "type": "",
+        "user": {},
+        "measurements": {},
+        "breakdowns": {},
+    }
+    project_id = row["project_id"]
+    event_id = row["event_id"]
+
+    event: Event = Event(project_id=project_id, event_id=event_id, data=data)
+    occurrence_data: IssueOccurrenceData = IssueOccurrenceData(
+        id="",  # TODO: need to generate this
+        project_id=project_id,
+        event_id=event_id,
+        fingerprint=_retrieve_fingerprints(event_id, project_id),
+        issue_title="",
+        subtitle="",
+        resource_id=None,
+        evidence_data="",
+        evidence_display="",
+        type="",
+        detection_time="",
+        level="",
+    )
+
+    return event, occurrence_data, int(row["group_id"])
+
+
+def _retrieve_fingerprints(event_id: str, project_id: int) -> Sequence[str]:
+    from sentry import eventstore
+
+    event = eventstore.get_event_by_id(project_id, event_id)
+    if event is None:
+        raise ValueError(f"no event found in eventstore({project_id}, {event_id})")
+
+    # We always fetch the stored hashes here.  The reason for this is
+    # that we want to show in the UI if the forced grouping algorithm
+    # produced hashes that would normally also appear in the event.
+    hashes = event.get_hashes()
+
+    # Transactions events are grouped using performance detection. They
+    # are not subject to grouping configs, and the only relevant
+    # grouping variant is `PerformanceProblemVariant`.
+
+    from sentry.utils.performance_issues.performance_detection import EventPerformanceProblem
+
+    problems = EventPerformanceProblem.fetch_multi([(event, h) for h in hashes.hashes])
```

Comment:
Probably worth returning this entire objects, since the `PerformanceProblem` maps reasonably well to our occurrence format.